payments/schema/mutations.py

```diff
@@ -379,6 +381,26 @@ def mutate_and_get_payload(cls, root, info, **input):
         return DeleteBerthProductMutation()
 
 
+class ConfirmPaymentMutation(graphene.ClientIDMutation):
```

Comment:
Should we also send some kind of `one-time-token` to the UI and then expect it on the `confirm` mutation just to have some kind of check that the request is not just coming from a random client?

Reply:
I think that the `order_number` already is our "one-time-token". If we are concerned that it's too easy to guess, then we should make `order_number` stronger instead. Are we concerned or has "respa" been concerned about it? :)

Reply:
Should be safe enough, just thinking on how to make it a bit more secure, but the ID generation is not a concern.

Reply:
good thing, that you thought about it 👍  I think `order_number` is strong enough for now and in the future (fopefully 😅 ), the customer will be paying these after logging into her profile, so there will be an auth layer

Reply:
good discussion. one clarification: there will be no customer login in the first version we are about to release.

Reply:
yes, but that is a feature mentioned on the roadmap of the project, so maybe hopefully potentially one day 🙂